lib/dealer.rb

```diff
@@ -0,0 +1,31 @@
+class Dealer
+
+  attr_reader :attributes
+
+  def initialize(attributes)
+    @attributes = attributes
+  end
+
+  def new_deck
+    cards.shuffle
+  end
+
+  def get_card(id)
+    cards[id]
```

Comment:
huh, am I misreading this perhaps? I see in your `Game` class you're calling `get_cards` with specific card ids (since `deck` and `board` are really just arrays of card ids right?). But `cards[id]` looks like a regular array index lookup.